### PR TITLE
Issue 65

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: grifts/seed.go [hunk 1]
```diff
@@ -37,6 +38,41 @@ var _ = Set("admin", func(c *Context) error {
 	return nil
 })
 
+var _ = Set("speaker", func(c *Context) error {
+	if len(os.Args) < 9 {
+		return fmt.Errorf("Usage: speaker [first_name] [last_name] [bio] [website] [twitter] [linkedin] [facebook] [contact_id] [photo_url]")
```
Comment (briandowns) [inline grifts/seed.go]: Do we want to care about the contact ID from the cli?

Comment (bketelsen) [reply grifts/seed.go]: I wouldn't think so

Comment (briandowns) [reply grifts/seed.go]: I didn't either but figured I'd be safe than sorry.

**Comment by bketelsen** [comment]
Thanks for this PR!

**Pull request merged**